ewcfg.py

```diff
@@ -3225,8 +3253,36 @@ def wef_bow(ctn = None):
 
 	elif aim >= (9 - int(16 * ctn.crit_mod)):
 		ctn.crit = True
-		ctn.slimes_damage = int(dmg * 10)
+		ctn.slimes_damage = int(dmg * 6)
+		
+# weapon effect function for "Dragon Claw"
 
+def wef_dclaw(ctn = None):
+	
+	user_mutations = ctn.user_data.get_mutations()
+	if mutation_id_fastmetabolsim in user_mutations or mutation_id_lightasafeather in user_mutations:
+                ctn.slimes_damage = int(ctn.slimes_damage * 1.2)
+                ctn.slimes_spent *= 0.5
+        else:
+                ctn.slimes_damage = int(ctn.slimes_damage * 1.5)
+                ctn.slimes_spent *= 1
+
+        #less slime cost and less damage = attacking faster I guess?
+	ctn.sap_damage = 5
+	ctn.sap_ignored = 10
+	if aim <= (random.randint(1,13):
```

Comment:
unmatched parenthesis. please at least run your code to make sure it works

Reply:
also aim is undefined

Reply:
the "aim" variable is still undefined

Reply:
May have accidentally changed that in the wrong file